processors/mapsme.py

```diff
@@ -39,24 +47,24 @@ def uid(elid, typ=None):
 class DummyCache:
     """This class may be used when you need to omit all cache processing"""
 
-    def __init__(self, cache_path, cities):
+    def __init__(self, cache_path: str, cities: list[City]) -> None:
         pass
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str) -> Any:
```

Comment:
It's not `Any`. It is `typing.Callable[..., None]`

<img width="592" alt="image" src="https://github.com/alexey-zakharenkov/subways/assets/93727145/1590aa78-902a-49a4-9baa-7fcfde15d224">

Reply:
\+